src/drivers/virtio/virtqueue/split.rs

```diff
@@ -60,7 +60,7 @@ impl AvailRing {
 		volatile_self: VolatilePtr<'_, Self, A>,
 	) -> VolatilePtr<'_, MaybeUninit<le16>, A> {
 		let ring_and_event_ptr = map_field!(volatile_self.ring_and_event);
-		ring_and_event_ptr.index(ring_and_event_ptr.len() - 2)
+		ring_and_event_ptr.index(ring_and_event_ptr.len() - 1)
```

Comment:
This reverses the change in the other commit, right? If so, I think we should amend the original commit.

Reply:
Oops